mitmproxy/addons/tlsconfig.py

```diff
@@ -616,7 +619,20 @@ def get_cert(self, conn_context: context.Context) -> certs.CertStoreEntry:
         # RFC 2818: If a subjectAltName extension of type dNSName is present, that MUST be used as the identity.
         # In other words, the Common Name is irrelevant then.
         cn = next((str(x.value) for x in altnames), None)
-        return self.certstore.get_cert(cn, altnames, organization)
+        return self.certstore.get_cert(cn, altnames, organization, revocation)
+
+    async def request(self, flow: http.HTTPFlow):
+        if not flow.live or flow.error or flow.response:
+            return
+        # Check if a request has a magic CRL token at the end
+        magic_token = str(self.certstore.default_ca.serial) + ".crl"
```

Comment:
This can be a `crl_path` `@property` that can be reused between `request` and `get_cert`.

Reply:
Should be resolved?

Reply:
Then you can just click resolve. :)

Reply:
Apologies, I wasn't sure if it is considered impolite to do so instead of letting the original starter of the conversation confirm it is indeed resolved and have them resolve it, will mark it as resolved though, ty!